test/wdio-ios.conf.js

```diff
@@ -21,10 +21,6 @@ config.capabilities = [
         // true, it runs our tests in the Cordova context.
         autoWebview: true,
 
-        // When set to true, it will not show permission dialogs, but instead grant all
-        // permissions automatically.
-        autoGrantPermissions: true,
```

Comment:
why did this go?

Reply:
http://appium.io/docs/en/writing-running-appium/caps/
-> android only